parser/scanner.l

```diff
@@ -459,6 +461,7 @@ sys_var		"@@"(({global}".")|({session}".")|{local}".")?{ident}
 {all}			return all
 {alter}			return alter
 {and}			return and
+{any}			return any
```

Comment:
Yes. ANY is unreserved keywords, Please add it to UnReservedKeyword list and add testcase in parser_test.go.
See: https://dev.mysql.com/doc/refman/5.7/en/keywords.html